src/utils.ts

```diff
@@ -294,11 +306,10 @@ export function buildOutputChunkWithCssInjectionCode(
     topExecutionPriorityFlag: boolean
 ): string {
     const appCode = jsAssetCode.replace(/\/\*.*empty css.*\*\//, '');
-    jsAssetCode = topExecutionPriorityFlag ? '' : appCode;
-    jsAssetCode += cssInjectionCode;
-    jsAssetCode += !topExecutionPriorityFlag ? '' : appCode;
 
-    return jsAssetCode;
+    return topExecutionPriorityFlag
+      ? `${cssInjectionCode}${appCode}` :
+      `${appCode}${cssInjectionCode}`;
 }
```

Comment:
Leave the code as before please

Reply:
sure, I can leave it as is. is my implementation wrong?